app/scripts/views/confirm_sign_in.js

```diff
@@ -0,0 +1,130 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define(function (require, exports, module) {
```

Comment:
Can you rename this file to `confirm_signin` to be consistent with the other file names? We standardized last year and had to update all references to `sign-in` and `sign_in`.